Content.Server/Chat/Systems/ChatSystem.Radio.cs

```diff
@@ -13,7 +13,33 @@ public sealed partial class ChatSystem
     /// Cache of the keycodes for faster lookup.
     /// </summary>
     private Dictionary<char, RadioChannelPrototype> _keyCodes = new();
-
+    //Dict of Headset IDs and what their "defaults" are
+    //probably should be put into the yml files themselves down the line
+    private Dictionary<string, string> headsetCommonDict = new()
```

Comment:
`_headsetCommonDict`

Reply:
Changed to using underscores in all private vars!